spec/appraisal/gemfile_spec.rb

```diff
@@ -14,7 +14,15 @@
     gemfile = Appraisal::Gemfile.new
     gemfile.source "one"
     gemfile.source "two"
-    expect(gemfile.to_s.strip).to eq %{source "one"\nsource "two"}
+    expect(gemfile.to_s.strip).to match %r{source "one"}
+    expect(gemfile.to_s.strip).to match %r{source "two"}
```

Comment:
Is there a reason why these two lines has to be changed? Is it the case now that the order are nondeterministic?

If so, I'd rather keep the array then `.uniq` it, as we can't be sure if user will be getting the same order every time they run `appraisal generate` or not, and it's annoying when you didn't touch anything, upgrade appraisal, then your `source` lines get swapped.